src/components/Login/Login.css

```diff
@@ -0,0 +1,93 @@
+.button {
+  background-color: #EC6607;
+  color: white;
+  padding: 2px 6px 2px 6px;
+  border: none;
+  cursor: pointer;
+  height: 51px;
+  width: 471px;
+  font-size: 32px;
+}
+
+.buttonCenter {
+    text-align: center;
+}
+
+.inputCenter{
+    text-align: center;
+}
+
+.inputCenter label {
+    font-size: 32px;
+}
+
+input[type=email], input[type=password] {
+  width: 471px;
+  padding: 2px 2px 2px 2px;
+  height: 51px;
+}
+
+input[type=email] {
+  font-size: 28px;
+  }
+
+  
+input[type=password] {
+  font-size: 28px;
+  }
+
+.emailText {
+  margin: 100px 0px 0px -389px;
+}
+
+.passwordText {
+    margin: 15px 0px 0px -333px;
+  }
+
+@media (max-width: 800px) {
+  .button {
+    background-color: #EC6607;
+    color: white;
+    padding: 2px 6px 2px 6px;
+    border: none;
+    cursor: pointer;
+    height: 46px;
+    width: 324px;
```

Comment:
Instead of defining button or input widths, it might be better to use components to take all space in the container or fit to content.

The container size is easier to define for different screen sizes than each subcomponent separately